### PR TITLE
Grafana Update

### PR DESCRIPTION
**Pull request by mcinerneym**
Grafana should now update even if a Pi errors or goes over a threshold and and updated the shell and service files to be more general

### CODE REVIEW

**Comment by evanho9** [comment]
Looks good! Tested by team Manchester

**Comment by nguyenvic** [comment]
Looks good and worked!

**Review by evanho9 — APPROVED** [review]

**Review by nguyenvic — APPROVED** [review]

File: monitoring/grafana.service [hunk 1]
```diff
@@ -4,8 +4,8 @@ After=network.target
 
 [Service]
 Type=simple
-WorkingDirectory=/usr/bin/piman/monitoring
-ExecStart=/usr/bin/piman/monitoring/grafana.sh
+WorkingDirectory=/home/usr/piman/monitoring
```
Comment (breed) [inline monitoring/grafana.service]: we need to agree on a location for this code! we have conflicting ideas. can you start a discussion?

File: monitoring/grafana.py [hunk 2]
```diff
@@ -15,21 +16,19 @@ def parse():
...
-            time = int(float(pi[0].split(' -')[0]) * 1000)
+            ts = datetime.strptime(pi[0].split(' -')[0], '%a %b %d %H:%M:%S %Y')
+            time = ts.timestamp() * 1000
             try:
-                cpu_load = float(pi[3].replace("CPU load: ", "").replace(" ", ""))
-                ram = float(pi[4].replace("RAM usage: ", "").replace(" ", ""))
-                disk_usage = float(pi[5].replace("Disk usage: ", "").replace(" ", ""))
-                pids = int(pi[6].replace("# of PIDs: ", "").replace(" ", ""))
-                temp = float(pi[7].replace("Temperature: ", "").replace(" Celsius", "").replace(" ", ""))
+                cpu_load = float(pi[-5].replace("CPU load: ", "").replace(" ", ""))
```
Comment (breed) [inline monitoring/grafana.py:29]: why are you using negative indices?

**Review by breed — COMMENTED** [review]
i'm not sure what changed here besides Celcius to F and OK to OK I'M READY

File: monitoring/grafana.py (hunk 2 above)
Comment (mcinerneym) [reply monitoring/grafana.py:29]: When the pi's go over the threshold, a message would be added to the log that breaks the grafana reporting. With the negative indices, that problem doesn't happen anymore.

File: monitoring/grafana.service (hunk 1 above)
Comment (mcinerneym) [reply monitoring/grafana.service]: A Canvas Discussion, right?

File: monitoring/grafana.py (hunk 2 above)
Comment (breed) [reply monitoring/grafana.py:29]: put a comment in to explain.

**Review by breed — CHANGES_REQUESTED** [review]

File: monitoring/grafana.service [hunk 3]
```diff
@@ -4,8 +4,8 @@ After=network.target
 
 [Service]
 Type=simple
-WorkingDirectory=/home/usr/piman/monitoring
-ExecStart=/home/usr/piman/monitoring/grafana.sh
+WorkingDirectory=/home/usr/local/share/piman/monitoring
```
Comment (breed) [inline monitoring/grafana.service]: these paths are incorrect.

**Review by breed — CHANGES_REQUESTED** [review]

**Review by breed — APPROVED** [review]
thanks

**Pull request merged**